marklogic-data-hub-central/ui/e2e/cypress/support/pages/graphExplore.tsx

```diff
@@ -27,7 +27,7 @@ class GraphExplore {
   }
 
 
-  getAllNodes() {
```

Comment:
@vmendez-marklogic I see we have renamed lot of functions as part of this ticket. And here we changed the name to get but seems like we are not just getting element here.